test.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+
+# This script builds and tests the site locally
+
+# Install the requirements for our site
+npm install
+
+# Start the server up
+npm run dev
```

Comment:
How would you know if the site is already running? Mayb you need to combine these last 2 conditionals and one is the check and the other is the action?

```
if ( curl site )
then
  site is up
else
  npm run dev
fi
```

Reply:
Checking if the site is running should not be coupled with starting the server, at least not in this way. The test is just for testing routes of the server and similar funtionality.